### DISCUSSION TITLE
How do I get the Pull Shark Badge?

### DISCUSSION THREAD

**Discussion by vickybelario** [community/community#138402] [Discussions] [answered]
### Select Topic Area

Question

### Body

what the specific steps or contributions required to receive it?

**Comment by itisrude** [comment]
Hello vickybelario,

The Pull Shark badge is given to those users who participate in the creation of pull requests and contribute towards projects on GitHub. The following is how one can get it:

### Pull Shark Badge - How to Get It

1. **Create PRs**:  
   Contribute to repositories. You do this by starting some issues or enhancements in the repository, then create a fork of that repository, make the own changes, and finally create a pull request for it.

2. **Valuable Pull Requests**:  
   Make sure your pull requests are of great value, hence contributing much to the project. It can be a bug fix, new feature, or some sort of optimization.

3. **Get Your Pull Requests Merged**:  
   More merged pull requests bring you closer to the badge. Make your pull request comply with the contribution guidelines of the repository and make sure that it is well-documented so that maintainers will have an easy time reviewing them.

4. **Keep Contributing**:  
   Consistency speaks. Continue contributing to various repositories; the badge isn't granted from one PR but from continued contributions over a period of time.

I suggest you look at this for a detailed guide on how to get the badge, if this doesn't help you I highly suggest you search it up on YouTube.
https://github.com/drknzz/GitHub-Achievements

Once GitHub notices that indeed you have reached the threshold for contributions with pull requests, you will automatically get the Pull Shark badge.

If you have more questions or if you want me to assist in making your first pull request, let me know!

**Comment by dhans11** [comment, marked as answer]
You can get this badge by merging 2 pull requests.
